frontend/src/components/SourceView.js

```diff
@@ -210,6 +209,7 @@ class SourceView extends Component {
     this.focusLine( index )
   }
 
+  // FIXME: @Harjot1Singh
```

Comment:
What needs fixing here?

Reply:
I just disabled eslint rule there and should be fixed sometime in future